data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/RestApiConstants.java

```diff
@@ -48,9 +48,9 @@
       "No content may be returned if the genesis block has not been set, meaning that there is no head to query.";
   public static final String INVALID_BODY_SUPPLIED = "Invalid body supplied.";
 
-  public static final String EPOCH_QUERY_DESCRIPTION = "`UnsignedLong` Epoch number to query.";
+  public static final String EPOCH_QUERY_DESCRIPTION = "`UInt64` Epoch number to query.";
   public static final String SLOT_QUERY_DESCRIPTION =
-      "`UnsignedLong` Slot to query in the canonical chain.";
+      "`UInt64` Slot to query in the canonical chain.";
```

Comment:
@rolfyone Do we want this to still say `UnsignedLong`?  I've changed all the places that openapi referred to `UnsignedLong` to `uint64` so its consistent with the `@Schema` annotations we have.